README.md

```diff
@@ -1,16 +1,439 @@
+```
+
+When you invoked a function, the global context was _implicitly_ passed.
+
+JavaScript does this so that by default our functions can access any properties
+on that most-global object, `window` (or `global` in Node). Consider:
+
+```js
+let locationReturner = function() {
+  return this.location.host
+}
+
+locationReturner() //=> URL host serving this page e.g. developer.mozilla.org
+```
+
+Doesn't that seem _nice_ and handy to do? In an effort to be nice to us
+programmers, JavaScript's implementers made the global object available inside
+all functions.
+
+> **NEUROLINGUISTIC/ANTHROPOLOGICAL ASIDE**: Some people think that `this` is a
+> strange thing to call such an important concept. But pronouns like "this,"
+> "he," or "here" all refer to a _context_. At a concert, if I say (scream)
+> "It's noisy here," you don't think "Here in the Milky Way galaxy? I disagree.
+> Space has little oxygen as a sound medium and is therefore quite quiet." To
+> the contrary, you recognize the most relevant context is at this significant
+> and unusual happening with giant speakers and guitar players and know that
+> "here" refers to "this concert." JavaScript thought the best pronoun to use
+> was `this`, and it seems sensible to us.
+
+This rule is ***very, very, very*** important and is at the root of a vital
+bug we'll explore in just a few lessons.
+
+It's worth noting that even in a function inside of another function, the inner
+function's default context is still the global object:
+
+```js
+
+function a() {
+  return function b() {
+    return this;
+  }
+}
+
+a()() === window //=> true
+
+### Prevent Implicitly Setting in Function Calls With `use strict`
+
+We wish we could say that the default context was **always** the global object.
+It'd make things simple.
+
+However, in JavaScript, if the engine sees the `String` "use strict" inside a
+function, it will _stop_ passing the implicit _execution context_ of the global
+object.  If JavaScript sees `"use strict"` at the top of a JavaScript code
+file, it will apply this rule (and other strict behaviors) to _all functions_.
+
+```js
+function looseyGoosey() {
+  return this
+}
+
+function noInferringAllowed() {
+  "use strict"
+  return this
+}
+
+looseyGoosey() === window; //=> true
+noInferringAllowed() === undefined //=> true
+```
+
+There are really no guidelines as to which you'll see more. Some programmers
+think `strict` stops confusing bugs (seems wise!); others think it's an obvious
+rule of the language and squelching it is against the language's love of
+functions (a decent argument!). Generally, we advise you to think of the
+"default mode" as the one that permits an _implicit_ presumption of context.
+For more on strict-mode, see the Resources.
+
+### Access Implicitly-Set Parent Object in a Contained Function Expression
+
+This (no pun intended) is simple, thankfully: functions inside Objects have
+`this` pointing to their container so that they can access other sibling
```

Comment:
Not exactly how this rule works - it's the 'thing to the left of the dot' that is passed as the context. Hence
```js
byronPoodle.warn()
// LOG: Byron issues an ear-rupturing atomic bark when he hears noises in the apartment hallway
w = byronPoodle.warn
w()
// issues an undefined when he hears undefined
```

So, this is misleading at best